LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright [2014-2019] [Heudiconv developers]
+Copyright [2014-2023] [HeuDiConv developers]
```

Comment:
```suggestion
Copyright [2014-2024] [HeuDiConv developers]
```

Reply:
note : this is nothing to do with tutorials -- should have been a separate PR which could review/merge quickly instead of bundling here